app/uk/gov/hmrc/homeofficesettledstatusstubs/stubdata/TestStubData.scala

```diff
@@ -468,6 +468,47 @@ object TestStubData extends DataSet {
       )
     )
   )
+  val SuzyWolf = StatusCheckResult(
+    "Suzy Wolf",
+    LocalDate.parse("15/10/1971", formatter),
+    "ESP",
+    List(
+      ImmigrationStatus(
+        statusStartDate = LocalDate.parse("20/10/2021", formatter),
```

Comment:
Will acceptance tests start failing in 6 months as these 'expire' ?

Reply:
No it will not fail.

Reply:
Not specifically this one, but in other stub data the tests do depend on, in 6 months these status will not be considered active, we may want a peice of work to make these dynamic, so that they always return with a certain number of active statues

Reply:
okay